docs/assets/images/favicons/site.webmanifest

```
@@ -0,0 +1,19 @@
+{
+  "name": "",
+  "short_name": "",
```

```suggestion
  "name": "Dialtone",
  "short_name": "Dialtone",
```

We should at the very least have these filled out for our site manifest

---

Maybe also look into adding back `description` and `lang` at some time